pkg/management/component.go

```diff
@@ -304,7 +326,12 @@ func (c *component) DeleteUser(ctx context.Context, realmName, userID string) er
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
 	err := c.keycloakClient.DeleteUser(accessToken, realmName, userID)
+	if err != nil {
```

Comment:
Would it be better to create a transaction, delete the user in DB, delete the user in KC and rollback/commit if there is/isn't error? (same question for CreateUser)

Reply:
That's good point, I will improve it.

Reply:
I think we need to discuss about transactions in general when we will be both back from holidays. So I will temporary postpone this improvement.